### PR TITLE
Added Flask and database Tests

### PR DESCRIPTION
**Pull request by jiyapatel4**
Hi Chizaram,

I created tests for the Flask application and database tests. I can confirm that every test passed. 

My only concern is that the tests aren't exhaustive enough or fail to follow proper syntax and standards- any feedback would be appreciated.

EDITS:
- Perhaps the merge conflict is because your code already contains tests you've written?
- I did some refactoring and changed my code for the malformed tests after reviewing the tests you wrote for my database.

### CODE REVIEW

File: tests/test_db.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+        test_db.connect()
+        test_db.create_tables(MODELS)
+    
+    def tearDown(self):
+        test_db.drop_tables(MODELS)
+        # close connection to database
+        test_db.close()
+
+    def test_timeline_post(self):
```
Comment (akshitadixit) [inline tests/test_db.py:26]: could test adding invalid/empty params in the fields of the post submission

File: tests/test_app.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+        # POST request with empty content
+        response = self.client.post("/api/timeline_post", data={"name": "John Doe", "email": "john@example.com", "content" :""})
+        assert response.status_code == 400
+        html = response.get_data(as_text=True)
+        assert "Invalid content" in html
+
+        # POST request with malformed email
+        response = self.client.post("/api/timeline_post", data={"name": "John Doe", "email": "invalid-email", "content": "Testing 1"})
+        # assert response.status_code == 400
```
Comment (chizy7) [inline tests/test_app.py]: Looks good, however the last test for a malformed email seems to have a commented out assertion for the response status code. If this test is meant to check the response to a malformed email, the assertion should be uncommented and the expected status code should be confirmed.

File: tests/test_app.py
Comment (chizy7) [inline tests/test_app.py:1]: Overall, this is a good start. There are just a few minor improvements that could be made, particularly regarding how the response data is checked. 
But overall great job.

File: tests/test_db.py
Comment (chizy7) [inline tests/test_db.py:1]: Overall, this is a good start to enhance the reliability of the database interaction.

**Review by chizy7 — COMMENTED** [review]
Great job on the testing. Minor improvements can help make it more robust but overall, this is a great start.

File: tests/test_app.py (hunk 2 above)
Comment (jiyapatel4) [reply tests/test_app.py]: You're right, I missed that. I uncommented the assertion and can confirm that it passes.

**Review by chizy7 — APPROVED** [review]
Looks good. Thanks for making the changes. I will approve the PR although it won't be merged since there is already a completed unit test. The approval is to check this off as completed before the PR is closed. 

Great job!

**Pull request closed without merging**